src/components/ebay-textbox/index.js

```diff
@@ -0,0 +1,31 @@
+const markoWidgets = require('marko-widgets');
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+function getInitialState(input) {
+    const classes = ['textbox__control'];
+    if (input.fluid) {
+        classes.push('textbox__control--fluid');
+    }
+    return {
+        classes,
+        htmlAttributes: processHtmlAttributes(input),
+        rootClass: input.class || '',
+        value: input.value,
+        tag: input.fluid ? 'div' : 'span',
+        textBoxTag: Boolean(input.multiline) ? 'textarea' : 'input',
```

Comment:
We're treating `textbox` as one word, so just try to stay consistent, as in `textboxTag` rather than `textBoxTag`.